doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md

```diff
@@ -227,7 +244,7 @@ and the ask to:
 
 The steps would be:
 
-1) Take a checkpoint of the config
+1) Take the content of config DB here and store it as a checkpoint
```

Comment:
As per my understanding, we change the config then the previous config becomes a checkpoint. But the current config,  after the recent change, that doesn't convert into a checkpoint right?.

Reply:
No, we don't create automatic checkpoints.

Creating a checkpoint is a manual process. the user has to take a checkpoint at any point in time, decided to rollback at any point it time. 

We might consider automatically taking checkpoints in the future for some operations, but that's out of the scope of this design document.